lambdas/status_reports/src/t4_lambda_status_reports/__init__.py

```diff
@@ -0,0 +1,296 @@
+import asyncio
+import datetime
+import functools
+import itertools
+import os
+import typing as T
+
+import aiobotocore.session
+import jinja2
+
+
+CANARIES_PER_REQUEST = 5
+
+AWS_REGION = os.getenv("AWS_REGION")
+assert AWS_REGION
+
+
+session = aiobotocore.session.get_session()
+
+
+def create_cfn():
+    return session.create_client("cloudformation", region_name=AWS_REGION)
```

Comment:
I think it gets region from `AWS_DEFAULT_REGION` env var set by lambda runtime, so it doesn't make sense to pass region explicitly.